tests/shell_call/Makefile

```diff
@@ -0,0 +1,13 @@
+include ../Makefile.tests_common
+
+USEMODULE += shell
+
+DISABLE_MODULE += auto_init
+
+# chronos is missing a getchar implementation
+BOARD_BLACKLIST += chronos
+
+include $(RIOTBASE)/Makefile.include
+
+test:
```

Comment:
This target is not needed anymore.

Reply:
anyhow, please merge the test into tests/shell. They're testing the same module, and tests are kinda expensive for the CI...